### ISSUE TITLE
Can i start with multiple config files

### ISSUE THREAD

**Issue by microwl43** [fstab/grok_exporter#68]
Can i start a single grok-exporter with multiple config files?
Eg.
 ./grok_exporter -config file1 file2 file3 file4 
or
./grok_exporter -config file1 -config  file2 -config  file3 -config  file4 

The reason i ask is i have 4 static files in which i want to monitor the output of a string. 
2 files i expect same string to exist and 2 files with 2 other strings
these are the strings
`*=INFO<string>=DETAIL:<string>=DETAIL`
`*=info`
`*=INFO`

I created 4 files to monitor each looking pretty much like 
```
global:
    config_version: 2
input:
    type: file
    path: <path>/server.xml
    readall: true # Read from the beginning of the file? False means we start at the end of the file and read only new lines.
grok:
    patterns_dir: ./patterns
    additional_patterns:
    - 'EXIM_MESSAGE [a-zA-Z ]*'
metrics:
    - type: counter
      name: uview_server_xml_trace_level
      help: traceSpecification in the uview server.xml
      match:  'traceSpecification=%{QUOTEDSTRING:LOGLEVEL}'
      labels:
        LOGLEVEL: '{{.LOGLEVEL}}'

```

Multifile seems only to read the first file. 

grok_exporter_build_info{branch="master",builddate="2019-04-08",goversion="go1.12.2",platform="linux-amd64",revision="e2ba841",version="0.2.7"} 1

**Comment by microwl43** [comment]
For multiple instances of grok-exporter to run on a vm we need
1: To define the port in the config .yml file. Default is 9144. For each subsequent instance of grok-exporter we need to define the port E.G.
```
...
server:
    port: 9145
```
2: We need to open the port
`iptables -I INPUT -p tcp -m tcp --dport 9145 -j ACCEPT`
3: To start multiple instances
```
/opt/exporters/grok_exporter/grok_exporter -config /opt/exporters/grok_exporter/example/1.yml &
/opt/exporters/grok_exporter/grok_exporter -config /opt/exporters/grok_exporter/example/2.yml &
```

**Comment by fstab** [comment]
Reading multiple log files will be implemented with #5. I'm working on it, the internal file tailer implementation is ready to handle multiple log files, but the log line processor and config are not there yet.
